src/main/java/de/tum/cit/aet/artemis/core/service/telemetry/TelemetrySendingService.java

```diff
@@ -53,29 +62,41 @@ public TelemetrySendingService(Environment env, RestTemplate restTemplate) {
     private String operatorAdminName;
 
     @Value("${info.contact}")
-    private String contact;
-
-    @Value("${artemis.telemetry.sendAdminDetails}")
-    private boolean sendAdminDetails;
+    private String operatorContact;
 
     @Value("${artemis.telemetry.destination}")
     private String destination;
 
+    @Value("${spring.datasource.url}")
+    private String datasourceUrl;
+
+    @Value("${artemis.continuous-integration.concurrent-build-size}")
```

Comment:
This value may not be defined when not using the `localci` or `buildagent` profiles and then result in a failure to start Artemis. The suggestion sets a default value in such cases.
```suggestion
    @Value("${artemis.continuous-integration.concurrent-build-size:0}")
```